### ISSUE TITLE
Activating extension streetsidesoftware.code-spell-checker failed due to an error: Cannot find module 'settings-viewer-next'

### ISSUE THREAD

**Issue by EugeniyMaksimov** [streetsidesoftware/vscode-spell-checker#2531]
Cannot load the extension 

Extension verision is 2.17.0
VS Code version is 1.75.1

```text
2023-02-20 18:44:02.696 [error] Activating extension streetsidesoftware.code-spell-checker failed due to an error:
2023-02-20 18:44:02.696 [error] Error: Cannot find module 'settings-viewer-next'
Require stack:
- c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
- 
	at Module._resolveFilename (node:internal/modules/cjs/loader:987:15)
	at i._resolveFilename (node:electron/js2c/utility_init:17:840)
	at Module._load (node:internal/modules/cjs/loader:832:27)
	at c._load (node:electron/js2c/asar_bundle:5:13343)
	at r._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:122:13360)
	at g._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:62400)
	at E._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:61793)
	at Module.require (node:internal/modules/cjs/loader:1059:19)
	at p (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:4:699)
	at 8098 (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:741430)
	at n (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854157)
	at 6574 (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:282757)
	at n (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854157)
	at c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854769
	at Object.<anonymous> (c:\Users\Maksimov.Eugeniy\.vscode\extensions\streetsidesoftware.code-spell-checker-2.17.0\packages\client\dist\extension.js:2:854795)
	at u._compile (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:4:1313)
	at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
	at Module.load (node:internal/modules/cjs/loader:1035:32)
	at Module._load (node:internal/modules/cjs/loader:876:12)
	at c._load (node:electron/js2c/asar_bundle:5:13343)
	at r._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:122:13360)
	at g._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:62400)
	at E._load (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:61793)
	at Module.require (node:internal/modules/cjs/loader:1059:19)
	at require (node:internal/modules/cjs/helpers:102:18)
	at Function.r [as __$__nodeRequire] (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js:5:101)
	at f.pb (c:\Users\Maksimov.Eugeniy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:117:63664)
	at async Promise.all (index 0)
```

**Comment by ili101** [comment]
Same, not loaded, reverted to v2.16.0

**Comment by Jason3S** [comment]
@EugeniyMaksimov and @ili101,

Thank you. I'll take a look.

**Comment by Jason3S** [comment]
Fixed in 2.17.1

**Comment by EugeniyMaksimov** [comment]
Thank you very much for quick fix!
